internal/clients/sources/sources_client.gen.go

```diff
@@ -1,7 +1,7 @@
 // Package SourcesApi provides primitives to interact with the openapi HTTP API.
 //
 // Code generated by unknown module path version unknown version DO NOT EDIT.
-package SourcesApi
+package sources
 
 import (
 	"bytes"
```

Comment:
Please do not make manual edits to generate files. Remove it from this patch, I will do a PR to fix this. It looks like generation does not work correctly even.

Reply:
FYI the PR is here, WIP: https://github.com/RHEnVision/provisioning-backend/pull/46